app/Repositories/Contracts/PostRepository.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: quantien
+ * Date: 11/4/18
+ * Time: 6:20 PM
+ */
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.NamespaceDeclaration.BlankLineAfter</br>**Severity**: error</br>**File**: app/Repositories/Contracts/PostRepository.php</br>**Line**: 9</br>There must be one blank line after the namespace declaration